src/diffusers/utils/testing_utils.py

```diff
@@ -830,6 +830,17 @@ def run_test_in_subprocess(test_case, target_func, inputs=None, timeout=None):
         test_case.fail(f'{results["error"]}')
 
 
+def check_qkv_fusion_matches_attn_procs_length(model, original_attn_processors):
```

Comment:
we don't need these functions in testing_utils, they are very specific and you can just copy paste into each tests